### PR TITLE
Update top-level README

### PR DESCRIPTION
**Pull request by bernardnormier**
Small updates to the top-level README.

### CODE REVIEW

**Review by externl — APPROVED** [review]
Looks good!

File: README.md [hunk 1]
```diff
@@ -90,20 +96,23 @@ As copyright owner, ZeroC can license Ice under different license terms, and off
 - Commercial or closed-source licenses
 
 If you license Ice under GPL v2, there is no license fee or signed license agreement: you just need to comply with the
-GPL v2 terms and conditions. See [ICE_LICENSE](ICE_LICENSE) and [LICENSE](LICENSE) for further information.
+GPL v2 terms and conditions. ZeroC also grants a few [exceptions](ICE_LICENSE) to the GPLv2 terms and conditions.
```
Comment (InsertCreativityHere) [inline README.md:99]: This is inconsistent with how we write `GPL v2` everywhere else in this section.
I prefer `GPLv2`, but if we're going to add it here, we should change the other uses to still be consistent.
Or change this one to match the othes.

File: README.md [hunk 2]
```diff
@@ -90,20 +96,23 @@ As copyright owner, ZeroC can license Ice under different license terms, and off
 - Commercial or closed-source licenses
 
 If you license Ice under GPL v2, there is no license fee or signed license agreement: you just need to comply with the
```
Comment (InsertCreativityHere) [inline README.md:98]: I know we still link to the GPLv2 license via the badge, but I think it'd be nice to still have a link here in this section.

**Review by InsertCreativityHere — APPROVED** [review]

**Review by ReeceHumphreys — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (bernardnormier) [reply README.md:99]: Fixed. It's now spelled "GPLv2".

**Pull request merged**